vaas-app/src/vaas/api/client.py

```diff
@@ -84,4 +87,10 @@ def read_until(self, match, timeout=None):
         # override timeout for case when socket is open but process not responding
         if timeout is None:
             timeout = settings.VARNISH_COMMAND_TIMEOUT
-        return varnish.VarnishHandler.read_until(self, match, timeout)
+        buffer = varnish.VarnishHandler.read_until(self, match, timeout)
+        if len(buffer):
+            return buffer
+        else:
```

Comment:
this else is meaningless ;) - simple raise is enough.

```
if len(buffer):
    return buffer
raise VarnishApiReadException(
    'Timeout during varnish.VarnishHandler.read_until(self, {}, {})'.format(match, timeout)
)
```

Reply:
Indeed it is unnecessary, I removed it in next commit.